### DISCUSSION TITLE
Tittle and Citeky

### DISCUSSION THREAD

**Discussion by VaderihgBU** [BryanJenksCommunity/FAQ#184] [Q&A] [answered]
<%*
	let title = "{{title}}";
	let date = tp.date.now("YYYY-MM-DD");
	await tp.file.rename(`& ${date} ${title}`);
_%>

This code does not work for me. It only puts the title in the code itself, but not in the file. It is kept with the citekey. 

![image](https://github.com/BryanJenksCommunity/FAQ/assets/109186590/b0f8a906-e389-4069-8d3f-1a42a4db01e3)

**Comment by tallguyjenks** [comment, marked as answer]
update the template with this chunk:

```
<%*
    let title = "{{title | replace(':', '') | replace('#', '') | replace('^', '') | replace('|', '') | replace('\[', '') | replace('\]', '') | replace('\\', '') | replace('/', '')}}";
    let date = tp.date.now("YYYY-MM-DD");
    await tp.file.rename(`& ${date} ${title}`);
_%>
```

some of the issues people have been having is likely related to protected characters in the paper's file name that obsidian isn't liking
